centreon/config.new/packages/messenger.yaml

```diff
@@ -0,0 +1,15 @@
+framework:
+    messenger:
+        transports:
+            sync: 'sync://'
+
+        buses:
+            command.bus:
+                middleware:
+                    - validation
```

Comment:
Would it be possible to include a transaction middleware to ensure atomicity of handler execution?

Reply:
<3